src/widgets/DisassemblerGraphView.cpp

```diff
@@ -524,6 +550,89 @@ GraphView::EdgeConfiguration DisassemblerGraphView::edgeConfiguration(GraphView:
     return ec;
 }
 
+bool DisassemblerGraphView::eventFilter( QObject *obj, QEvent *event )
+{
+    if (Config()->getGraphPreview() && event->type() == QEvent::Type::ToolTip) {
```

Comment:
Better swap this, so the heavier operation is short-circuited.
```suggestion
    if (event->type() == QEvent::Type::ToolTip && Config()->getGraphPreview()) {
```